From e81974c4ebdca8a7a84b87a04bc70c60d9c1e48e Mon Sep 17 00:00:00 2001
From: "haotian.qu" <haotian.qu@amlogic.com>
Date: Thu, 6 Jul 2023 17:22:43 +0800
Subject: [PATCH 02/10] Autotest: Autotest supprt OTA case [1/1]

PD#SWPL-130834

Problem:
Autotest support OTA case.

Solution:
Add urlmisc cmd to mark misc partition to enter
recovery after reboot.

Verify:
local

Signed-off-by: haotian.qu <haotian.qu@amlogic.com>
Change-Id: Ic7254a04238b927ed4d340e9265612fe2d25638d
---
 recipes-core/images/amlogic-yocto.bb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/recipes-core/images/amlogic-yocto.bb b/recipes-core/images/amlogic-yocto.bb
index 6cae6d9c..be6f3746 100644
--- a/recipes-core/images/amlogic-yocto.bb
+++ b/recipes-core/images/amlogic-yocto.bb
@@ -21,7 +21,7 @@ IMAGE_INSTALL += " \
     ${@bb.utils.contains('DISTRO_FEATURES', 'selinux', 'packagegroup-core-selinux auditd', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'aml-bluez-alsa', 'aml-bluez-alsa', '', d)} \
     ${@bb.utils.contains("DISTRO_FEATURES", "gst1-plugins-tsplayer", "gst1-plugins-tsplayer", "", d)} \
-    ${@bb.utils.contains('DISTRO_FEATURES', 'swupdate', 'swupdate', '', d)} \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'swupdate', 'swupdate aml-bootloader-message', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'test_tools', 'test-tools', '', d)} \
     "
 
-- 
2.25.1

